nni/retiarii/experiment/pytorch.py

```diff
@@ -257,16 +259,30 @@ def stop(self) -> None:
         self._dispatcher_thread = None
         _logger.info('Experiment stopped')
 
-    def export_top_models(self, top_n: int = 1):
+    def export_top_models(self, top_k: int = 1, optimize_mode: str = 'maximize', formatter: str = 'code') -> Any:
```

Comment:
it is better to obtain `optimize_mode` from strategy

Reply:
This requires re-designing of strategy as some strategy does not need to know what mode it is by design (e.g., random). I'll add a comment that `optimize_mode` is likely to be removed and defined in strategy in future.